tests/unit/packet/PublishTest.php

```diff
@@ -15,134 +16,135 @@ public function testPublishStandard()
         $this->assertEquals(3, Publish::getControlPacketType());
     }
 
+    public function testExceptionIsThrownForUnexpectedPacketType()
```

Comment:
Adding this test was the only reason I noticed that `ControlPacket::checkRawInputValidControlPackageType()` wasn't working.